contracts/contracts/keeper/Keeper.sol

```diff
@@ -0,0 +1,194 @@
+pragma solidity 0.5.11;
+
+pragma experimental ABIEncoderV2;
+
+import "hardhat/console.sol";
+
+/**
+ * @title OUSD Keeper Contract
+ * @notice The Keeper is used to perform maintenance on vaults
+ * @author Origin Protocol Inc
+ */
+
+import { IKeeper } from "../interfaces/IKeeper.sol";
+import "../vault/VaultCore.sol";
+import "../token/OUSD.sol";
+import "../governance/Governable.sol";
+
+contract Keeper is IKeeper, Governable {
+    VaultCore vaultCore;
+    OUSD oUSD;
+
+    string CHECK = "check";
+    string EXECUTE = "execute";
+
+    string ALLOCATE = "allocate";
+    string REBASE = "rebase";
```

Comment:
Strings can be constants I think

Reply:
yep that came up in the `slither` - resolved